src/Result.rei

```diff
@@ -16,6 +16,10 @@ let ap: ('a, result('b, 'a => 'c)) => result('b, 'c);
 
 let map: ('a => 'b, result('c, 'a)) => result('c, 'b);
 
+let map2:
```

Comment:
map 2 should have the following type:
```reason
let map2: (('a, 'b) => 'c, result('d, 'a), result('d, 'b)) => result('d, 'c)
```

Reply:
The requested change has been made in the latest commit.